tests/misc/trunner_thirdparty.nim

```diff
@@ -0,0 +1,34 @@
+discard """
+  targets: "c cpp"
+  joinable: false
+"""
+
+#[
+runs tests that depend on 3rd party code and requires special treatment.
```

Comment:
Do you mean "3rd party code" or remote networking?

Reply:
good point, renamed trunner_thirdparty => trunner_special and reworded this paragraph to reflect the scope